guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,15 +4,45 @@
 [id="foreman-headline-features"]
 == Headline Features
 
-There are no highlights with Foreman {ProjectVersion}.
+=== Greatly decreased JavaScript size for plugins
+
+Previously Foreman's whole JavaScript bundle was duplicated in every plugin.
+Now a separate bundle is generated that Foreman and each plugin can reuse.
+In most plugins we saw a 2 to 3 MB reduction in size, so depending on the number of plugins this can easily save a lot.
```

Comment:
```suggestion
In most plugins, we saw a 2 to 3 MB reduction in size and depending on the number of plugins, this can save a significant amount of space.
```

Reply:
I'd say it's not about saving space. In my understanding space is something that you take up on disk. That misses the point because it's about the transfer size and amount of processing required when loading the page.